src/tests/ftest/util/server_utils.py

```diff
@@ -774,19 +774,22 @@ def update_config_file_from_file(self, dst_hosts, test_dir, generated_yaml):
         engines = generated_yaml["engines"]
         for i, engine in enumerate(engines):
             self.log.info("engine %d", i)
-            self.log.info("scm_mount = %s", engine["scm_mount"])
-            self.log.info("scm_class = %s", engine["scm_class"])
-            self.log.info("scm_list = %s", engine["scm_list"])
-
-            per_engine_yaml_parameters =\
-                DaosServerYamlParameters.PerEngineYamlParameters(i)
-            per_engine_yaml_parameters.scm_mount.update(engine["scm_mount"])
-            per_engine_yaml_parameters.scm_class.update(engine["scm_class"])
-            per_engine_yaml_parameters.scm_size.update(None)
-            per_engine_yaml_parameters.scm_list.update(engine["scm_list"])
-
-            self.manager.job.yaml.engine_params.append(
-                per_engine_yaml_parameters)
+            for storage_tier in engine["storage"]:
+                if storage_tier["class"] is not "dcpm":
```

Comment:
(pylint-literal-comparison) Comparison to literal

Reply:
This is a valid problem and should be fixed. In python, `==` is an equality test (values are the same), whereas `is|is not` is an identity test (is the same object, i.e. the same reference to memory).